projects/configs/tag-operations/mr-spectroscopy.yaml

```diff
@@ -0,0 +1,206 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+- name: Patient ID  # remove this before merging - useful for testing anonymisation as an 'external user', but 'op' should be 'secure-hash' as in base.yaml instead of 'replace'
+  group: 0x0010
+  element: 0x0020
+  op: replace
+- name: Acquisition DateTime
+  group: 0x0008
+  element: 0x002A
+  op: keep
+- name: Content Date
+  group: 0x0008
+  element: 0x0023
+  op: keep
+- name: Content Time
+  group: 0x0008
+  element: 0x0033
+  op: keep
```

Comment:
these don't feel like they're specific to mr spectroscopy, or am I wrong about that? 

I think if we haven't defined them in the base.yaml then probably should remove from here

Reply:
I think if they're optional but still count for that modality then I'd put them in MR at least. Though we should have the default to be replace for these